modules/bloxone_infra_host_gcp/userdata.tftpl

```diff
@@ -0,0 +1,7 @@
+#cloud-config
+host_setup:
+  jointoken: ${join_token}
+  tags:
+  %{ for k, v in tags ~}
```

Comment:
why the `~` at the end here, I don't see it in AWS and Azure module

Reply:
"~" removes the addition line that gets added after each "tag"  value